low-level/files/opt/seagate/sspl/conf/sspl.conf.LR2

```diff
@@ -204,7 +204,7 @@ sspl_log_file_path=/var/log/cortx/sspl/sspl.log
 operating_system=centos7
 kernel_version=
 log_level=INFO
-product=LDR_R2
+product=LR2
```

Comment:
why this change? Files also renamed for same!
INI files we need to remove eventually. Also product needs to be in global config.